### PR TITLE
Update wheeler environments to load Python

### PR DESCRIPTION
**Pull request by osheamonn**
We now require NumPy 1.10 which is greater than the wheeler system NumPy version, so load the module on wheeler which satisfies this requirement (loads NumPy 1.13)

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: support/Environments/wheeler_gcc.sh [hunk 1]
```diff
@@ -26,6 +26,7 @@ spectre_unload_modules() {
     module unload git/2.8.4
     module unload lcov/1.13
     module unload charm/6.8.0-smp
+    module load python/anaconda2-4.1.1
```
Comment (wthrowe) [inline support/Environments/wheeler_gcc.sh]: unload

**Review by wthrowe — CHANGES_REQUESTED** [review]

File: support/Environments/wheeler_gcc.sh (hunk 1 above)
Comment (osheamonn) [reply support/Environments/wheeler_gcc.sh]: Indeed

**Review by mar-celine — APPROVED** [review]

**Review by wthrowe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Comment by kidder** [comment]
@nilsdeppe please look at

**Review by nilsdeppe — APPROVED** [review]

**Comment by nilsdeppe** [comment]
travis failures are timeouts

**Pull request merged**